communities.md

```diff
@@ -8,12 +8,13 @@ layout: black
 <h1>FAIR Catalogues</h1>
 Make data and samples from biobanks, registries and research collections findable to ensure maximum (re)use for research and improve patients and citizens lives. Examples:
 <ul>
+<li><a href="data-catalogue.molgeniscloud.org">EU health data networks and harmonization catalogue</a></li>
 <li><a href="http://directory.bbmri-eric.eu">European directory of biobank collections (BBMRI-ERIC)</a></li>
 <li><a href="http://catalogue.bbmri.nl">Dutch catalogue of human data and sample collections (BBMRI-NL)</a></li>
-<li><a href="http://catalogue.lifecycle-project.eu/">EU child network for multi-center cohort harmonisation</a></li>
+<li><a href="https://umcgresearchdatacatalogue.nl/">University Medical Center Groningen Cohorts and Data Catalogue (UMCG)</a></li>
 <li><a href="http://samples.rd-connect.eu">Catalogue of rare disease samples (RD-Connect)</a></li>
 <li><a href="http://www.palgaopenbaredatabank.nl/">Dutch catalogue of pathology samples (PALGA)</a></li>
```

Comment:
I came here out of curiousity, but since I'm here I might as well be bold enough to place a comment:

It's probably only documentation, but I think it's better to change these links to HTTPS, as these also seem to work, same for the ones above.

Reply:
yes, this is pretty old stuff